### ISSUE TITLE
magic prompt limited edition ?

### ISSUE THREAD

**Issue by yami-no-tusbas** [adieyal/sd-dynamic-prompts#40]
Here is my prompt : 
```solo full body girl, intricate detailed realistic ultra detailed green eyes, intriced (long) [white|black] hair, very detailed [black|red] [short dress|bikini],choker, boots,
on stage, club, indoor, table, seats, chair, bar, standing microphone, realistic lighting, colored lighting,
highly detailed, digital painting, concept art, smooth, sharp focus, sharp focus, illustration, hyperdetailed, hyperreal, trending on artstation
ilya kuvshinov and range murata,
```

Here is what is send to the generator : 
```solo full body girl, intricate detailed realistic ultra detailed green eyes, intriced (long) [white|black] hair, very detailed [black|red] [short dress|bikini],choker, boots,
on stage, club, indoor, table, seats, chair, bar, standing microphone, realistic lighting, colored lighting,
highly detailed, digital painting, concept art, smooth, sharp focus, sharp focus, illustration, hyperdetailed, hyperreal, trending on artstation
ilya kuvshinov and range murata, holding
```

The only thing added is "holding"... I had that multiple times, seems like the longer the prompt is, the less magic prompt works. Maybe because of some arbitrary limitation, but this is annoying really fast.

Here is the PNG if  you want to extract the data from it. <https://i.imgur.com/cJVCyAG.png>

I wish we can enable the plugin to have no regards about the prompt lenght or something, maybe the plugin can have some sort of button you clic and it give you random parameters that you can choose to add to your prompt ?

**Comment by crackfoo** [comment]
yeah there does seem to be a text limit to the magic. Seems it won't apply magic past 150 charas

**Comment by StewieGriffin** [comment]
> 

I can confirm this, happening to me as well

**Comment by adieyal** [comment]
That's pretty much how it works. It takes a prompt and adds a bunch of modifiers. It seems to always prefix the output with the input prompt. Have a look here to see how it works: https://huggingface.co/spaces/Gustavosta/MagicPrompt-Stable-Diffusion

What output are you expecting?

**Comment by yami-no-tusbas** [comment]
> What output are you expecting?

Well... the one non truncated, in my exemple only on word was added to the prompt because it is a long one from the start.

I tried the online on and it works better than the plugin one. 

on stage, club, indoor, table, seats, chair, bar, standing microphone, realistic lighting, colored lighting, highly detailed, digital painting, concept art, smooth, sharp focus, sharp focus, illustration, hyperdetailed, hyperreal, trending on artstation ilya kuvshinov and range murata, `alphonse mucha, artgerm, adolphe bouguereau, craig mullins, j. c. leyendecker, rene maritte, tian zi, wlop, alsphonse mucha, artgerm`

and with the plugin in automatic1111 gui's :

on stage, club, indoor, table, seats, chair, bar, standing microphone, realistic lighting, colored lighting, highly detailed, digital painting, concept art, smooth, sharp focus, sharp focus, illustration, hyperdetailed, hyperreal, trending on artstation ilya kuvshinov and range murata, `holding`
